tests/build_/test_build_request.py

```diff
@@ -1105,14 +1138,43 @@ def test_render_orchestrate_build(self, platforms, secret, disabled,
                                             'build_kwargs')
             assert build_kwargs['arrangement_version'] == arrangement_version
 
+            worker_config_kwargs = plugin_value_get(plugins, phase, plugin, 'args',
+                                                    'config_kwargs')
+
+            dummy_conf_file = os.path.join(str(tmpdir), 'dummy-osbs.conf')
+            worker_config = Configuration(conf_file=dummy_conf_file, **worker_config_kwargs)
```

Comment:
`conf_file=None` would do here wouldn't it? Maybe there's something I'm missing.